Description:  Remove sslv3 support to fix build with new openssl (Closes: 804604).
Author: Peter Michael Green <plugwash@raspbian.org>
Bug-Debian: https://bugs.debian.org/804604

--- fetchmail-6.3.26.orig/socket.c
+++ fetchmail-6.3.26/socket.c
@@ -914,7 +914,8 @@ int SSLOpen(int sock, char *mycert, char
 			return -1;
 #endif
 		} else if(!strcasecmp("ssl3",myproto)) {
-			_ctx[sock] = SSL_CTX_new(SSLv3_client_method());
+			report(stderr, GT_("Your operating system does not support SSLv3.\n"));
+			return -1;
 		} else if(!strcasecmp("tls1",myproto)) {
 			_ctx[sock] = SSL_CTX_new(TLSv1_client_method());
 		} else if (!strcasecmp("ssl23",myproto)) {
